tests/generic_config_updater/patch_sorter_test.py

```diff
@@ -1741,6 +1790,26 @@ def test_sort__replacing_create_only_field__success(self):
         # Assert
         self.assertNotEqual(None, actual)
 
+    def test_sort__adding_create_only_field__success(self):
```

Comment:
In a future PR, going to test for the actual steps like suggested by Renuka. This is going to help with the nightly tests.